meetings/2022-04-04.md

```diff
@@ -0,0 +1,103 @@
+* SC: Besides editor's draft, CG-Draft is fine, which is currently what's in the PR. There are specs in the notifications panel that can be used as an example too.
+* AC: Note that there have been some reviews on the primer, with improvements suggestions, some of which have been merged already. The approach we can take is to make changes in the source document. For more substancial changes, we can open issues in the source repository first.
+* eP: +1 on inline changes of the source document. Tracking feedback in issues is also a good thing. How do we update the . We should reconsider having hard dates in the snapshot not to be prevented from editing the PRs proposal
+* SC: One approach that could work (currently outside of a process), instead of PRing into solid-specifications, would be to get the editors to come to the source repositories (to review the PR). All the feedback iteration happens in the source repository, and then once it is accepted getting the latest version to the solid-spec repo is just a mecanical thing. Reviewers should go to the source, instead of the proposals coming to the solid-spec.
+* AC: On the one hand, having PRs going to the spec repo can be useful for gating, but +1 on reviews happening in the editor's draft in tghe source repo. Change is easier to handle there, so it would be ideal. Realistically though, editors won't necessarily go to the source repos until a PR is opened on the solid-spec repo
+* eP: Editors have editor's meeting, so we could signal the fact that new versions of the editor's draft is about to be published so that they can take action to review it. Reviews don't need to happen in a PR, but rather in the snapshot that is the draft once the PR has been merged.
+* SC: +1. The notification panel right now branched the snapshots for inclusion to be included in the solid-specification repository. When a WG proposes a document be published, the reviewers review the snapshot to be added, and not the draft. Then the editor's draft is updated, and a new snapshot is created, and so on. No strong feeling about which repository should be reviewed. We don't want reviews to become unclear because it's unclear what reviewers are committing to.
+* AC: Right now we make a PR to the solid-spec, that's the snapshot. We can continue like this. There is an issue with automation though, because updating the draft to become a snapshot is very manual and error-prone. I've put together some automation to create a snapshot in a branch based on a tag. The challenge is with the first TR: there is currently nothing at the TR's URL. As soon as we merge something into this locaion, we can refer to it, but to be able to refer to it in the snapshot it needs to be created. If we merge the current snapshot in the solid-spec repo, a document will exist at the TR URL, but itself will refer to a non-existing document. Is it something all TR's face ?
```

Comment:
```suggestion
* AC: Right now we make a PR to the solid-spec; that's the snapshot. We can continue like this. There is an issue with automation though, because updating the draft to become a snapshot is very manual and error-prone. I've put together some automation to create a snapshot in a branch based on a tag. The challenge is that with the first TR, there is currently nothing at the TR's URL. As soon as we merge something into this location, we can refer to it, but to be able to refer to it in the snapshot, it needs to be created. If we merge the current snapshot in the solid-spec repo, a document will exist at the TR URL, but will itself refer to a non-existing document. Is this something all TRs face?
```